scripts/functional-tests.sh

```diff
@@ -64,10 +85,10 @@ ilab --version
 ilab sysinfo
 
 # pipe 3 carriage returns to ilab config init to get past the prompts
-echo -e "\n\n\n" | ilab config init
+echo -e "\n\n\n" | ilab init
```

Comment:
Why reverting this?

Reply:
`ilab config init` was previously protected by a config-guard which ensured that the `config` command group doesn't actually have a config object sent to it. Since this is something we need to change to view the config with `ilab config show`, I removed the `config` group from the no-config guard. So we need to call `ilab init` directly instead of having it be under config.